presto-clp/src/main/java/com/yscope/presto/split/ClpMySQLSplitProvider.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yscope.presto.split;
+
+import com.facebook.airlift.log.Logger;
+import com.facebook.presto.spi.SchemaTableName;
+import com.google.common.collect.ImmutableList;
+import com.yscope.presto.ClpConfig;
+import com.yscope.presto.ClpSplit;
+import com.yscope.presto.ClpTableLayoutHandle;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ClpMySQLSplitProvider
```

Comment:
The capitlization of this name should probably be `ClpMysqlSplitProvider`; if you were creating a variable starting with the word MySQL, it would be `mysqlSplitProvider` rather than `mySQLSplitProvider`.